packages/core/src/scss/components/checkout/loadingSkeleton/_loadingSkeleton.scss

```diff
@@ -0,0 +1,215 @@
+@keyframes lightColorTransition {
+    0%, 100% {
+        background-color: $loading-skeleton-grey-box-color;
+    }
+    50% {
+        background-color: $loading-skeleton-grey-box-color-2;
+    }
+}
+
+.checkout-page-skeleton div,
+.address-form-skeleton div,
+.checklist-skeleton div,
+.order-confirmation-page-skeleton div,
+.walletbuttons-skeleton div{
+    background: $loading-skeleton-grey-box-color;
+    border-radius: $global-radius;
+    content: ' ';
+    display: block;
+    height: $floating-label-input-height;
+    margin-bottom: spacing("single");
+    animation: $loading-skeleton-grey-box-animation;
+}
+
+.walletbuttons-skeleton div {
+    height: $wallet-button-height;
+    margin-bottom: 0;
+}
+
+.order-confirmation-page-skeleton {
+    .thankyou {
+        margin-top: spacing("double");
+        width: 38.2%;
```

Comment:
Hard code some values as they are not reused and mainly for making the grey box shapes.